plugin/src/main/scala-2/org/scalafmt/sbt/CredentialsUtils.scala

```diff
@@ -0,0 +1,7 @@
+package org.scalafmt.sbt
+
+import sbt.librarymanagement.ivy.{Credentials, DirectCredentials}
```

Comment:
this is unlikely to pass the formatting check.

Reply:
Will run `scala-cli fmt .` and `scala-cli fmt --check` before force push